index.ts

```diff
@@ -577,16 +602,19 @@ function wktToGeoJSONinner(wktParser: WktParser, userOptions: WktUserOptions) {
  *   stringified as "EMPTY" WKT geometries. This is technically
  *   valid, but rare, for GeoJSON.
  */
-export function geoJSONToWkt(geometry: Geometry): string {
+export function geoJSONToWkt(
+  geometry: Geometry,
+  options: WktStringifyOptions = { version: "1.1.0" },
```

Comment:
```suggestion
  options: WktStringifyOptions = { version: "1.2.0" },
```
What do you think about making it default? Would require a breaking change version.

Reply:
I think I'd rather add the option first and make it default once there other changes that justify a major version. Given NPM behavior, I really want to keep on the same major version for a while because otherwise people don't get bugfixes.